internal/pkg/fixtures/fixtures.go

```diff
@@ -97,15 +97,27 @@ func (r *ConfigRow) ToModel() *model.ConfigRow {
 	return row
 }
 
-func (b *BranchConfigs) GetName() string {
-	return b.Branch.Name
+func (b *Branch) String() string {
+	return b.Description
 }
 
-func (c *Config) GetName() string {
+func (c *Config) String() string {
+	return c.Description
```

Comment:
Aby bolo mozne objekty zoradit, musia mat definovane `ObjectName` a `String` metody.
V tychto `fixtures` nemam ziadne ID, preto tu dam `description`, tu je to jedno.